plugins/outputs/file/file_test.go

```diff
@@ -91,6 +145,62 @@ func TestFileExistingFiles(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestNewGzipCompressedFiles(t *testing.T) {
+	s := &influx.Serializer{}
+	require.NoError(t, s.Init())
+
+	fh1 := tmpFile(t)
+	fh2 := tmpFile(t)
+	fh3 := tmpFile(t)
+	f := File{
+		Files:                []string{fh1, fh2, fh3},
```

Comment:
Is there any particular reason you are testing this with 3 files?

Reply:
I think that's how it's done in another test I copy pasted it from.